### PR TITLE
small improves

### PR DESCRIPTION
**Pull request by tshepang**
Also, RUST_BACKTRACE=full is now default

### CODE REVIEW

File: src/diagnostics.md [hunk 1]
```diff
@@ -273,18 +273,19 @@ There are two main ways to find where a given error is emitted:
...
   will treat the first error being emitted as an Internal Compiler Error, which
-  allows you to use the environment variable `RUST_BACKTRACE=full` to get a
+  allows you to get a
   stack trace at the point the error has been emitted. Change the `1` to
-  something else if you whish to trigger on a later error. Some limitations
+  something else if you whish to trigger on a later error. One limitation
   with this approach is that some calls get elided from the stack trace because
-  they get inlined in the compiled `rustc`, and the same problem we faced with
+  they get inlined in the compiled `rustc`.
+  Another limitation is the same problem we faced with
```
Comment (JohnTitor) [inline src/diagnostics.md:286]: IMO: I feel something like the below is slightly better:
```
This approach has some limitations. First, ... Second, ...
```

How about?

File: src/diagnostics.md [hunk 2]
```diff
@@ -273,18 +273,19 @@ There are two main ways to find where a given error is emitted:
...
   constructed behind a relatively deep call-stack. Even then, it is a good way
   to get your bearings.
-- Invoking `rustc` with the nightly-only flag `-Z treat-err-as-bug=1`, which
+- Invoking `rustc` with the nightly-only flag `-Z treat-err-as-bug=1`
   will treat the first error being emitted as an Internal Compiler Error, which
-  allows you to use the environment variable `RUST_BACKTRACE=full` to get a
+  allows you to get a
   stack trace at the point the error has been emitted. Change the `1` to
-  something else if you whish to trigger on a later error. Some limitations
+  something else if you whish to trigger on a later error. One limitation
```
Comment (JohnTitor) [inline src/diagnostics.md:283]: Pre-existing but:

```suggestion
  something else if you wish to trigger on a later error. One limitation
```

File: src/diagnostics.md (hunk 1 above)
Comment (tshepang) [reply src/diagnostics.md]: I restructured it a bit, so the paragraph is less heavy

**Review by JohnTitor — APPROVED** [review]
Looks great, thanks!

**Pull request merged**